modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -276,6 +282,58 @@ private ObjectRelationship _testAddSystemObjectRelationship(String type)
 			StringUtil.randomId(), type);
 	}
 
+	private void _testSystemObjectRelationshipOneToMany() throws Exception {
+		ObjectRelationship objectRelationship;
+
+		String objectFieldNamePrefix;
+
+		for (String deletionType :
+				Arrays.asList(
+					ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+					ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+					ObjectRelationshipConstants.DELETION_TYPE_DISASSOCIATE)) {
+
+			objectRelationship =
+				_objectRelationshipLocalService.addObjectRelationship(
+					TestPropsValues.getUserId(),
+					_objectDefinition1.getObjectDefinitionId(),
+					_systemObjectDefinition.getObjectDefinitionId(),
+					deletionType,
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString()),
+					StringUtil.randomId(),
+					ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+
+			objectFieldNamePrefix = "r_" + objectRelationship.getName() + "_";
+
+			Assert.assertTrue(
+				_hasColumn(
+					_systemObjectDefinition.getExtensionDBTableName(),
+					objectFieldNamePrefix +
+						_objectDefinition1.getPKObjectFieldName()));
+			Assert.assertNotNull(
+				_objectFieldLocalService.fetchObjectField(
+					_systemObjectDefinition.getObjectDefinitionId(),
+					objectFieldNamePrefix +
+						_objectDefinition1.getPKObjectFieldName()));
+
+			objectRelationship =
+				_objectRelationshipLocalService.updateObjectRelationship(
+					objectRelationship.getObjectRelationshipId(),
+					objectRelationship.getDeletionType(),
+					LocalizedMapUtil.getLocalizedMap("Baker"));
+
+			Assert.assertEquals(
+				LocalizedMapUtil.getLocalizedMap("Baker"),
+				objectRelationship.getLabelMap());
+
+			_objectRelationshipLocalService.deleteObjectRelationship(
+				objectRelationship);
+
+			Assert.assertFalse(_hasTable(objectRelationship.getDBTableName()));
```

Comment:
@p-albuquerque This Assert is just for MANY_TO_MANY relationships. Can you remove this code? Thx

Reply:
@p-albuquerque Here you should make something like this:

```
Assert.assertNull(
			_objectFieldLocalService.fetchObjectField(
				_objectDefinition2.getObjectDefinitionId(),
				objectFieldNamePrefix +
					_objectDefinition1.getPKObjectFieldName()));
```

Reply:
Because in this case a field is created (not a table). If you have any doubt about it, just reach me :D